src/ui/ui_manager.py

```diff
@@ -586,14 +586,23 @@ def buy_pots(self, healing_pots: int = 0, mana_pots: int = 0):
         :param healing_pots: Number of healing pots to buy
         :param mana_pots: Number of mana pots to buy
         """
-        h_pot = self._template_finder.search_and_wait("SUPER_HEALING_POTION", roi=self._config.ui_roi["vendor_stash"], time_out=3)
+        if self._config.general["difficulty"].upper() == 'NIGHTMARE':
+            healingpotion = "GREATER_HEALING_POTION"
+            manapotion = "GREATER_MANA_POTION"
+        elif self._config.general["difficulty"].upper() == 'HELL':
+             healingpotion = "SUPER_HEALING_POTION"
+             manapotion = "SUPER_MANA_POTION"
+        else:
+             healingpotion = "HEALING_POTION"
+             manapotion = "MANA_POTION"
```

Comment:
I think there are no templates for "normal" potions yet. Did you maybe forget to add them?

Reply:
deleted this entry and just alowed super and greater potions.